src/weekly/weekly.service.ts

```diff
@@ -417,7 +417,8 @@ export class WeeklyService {
     const cred = await this.getGoogleCredentials();
     const oAuth2Client = await auth.authorize(cred);
 
-    const message = 'Timesheets: https://timesheets.opennetworking.org/\n\nPlease complete timesheet week: ' + week.begin + ' - ' + week.end;
+    const message = 'Approver requests the below:\n' +
```

Comment:
I'm not sure about the wording, but can't suggest anything better - approved.
Also Ain did want to have customizable/randomizable mails. Maybe that can be done in a separate PR. 
It might pop up a dialog when he clicks Reminder, asking him to enter a comment - and this can be used in the email **Subject**